bin/server/cmd/root.go

```diff
@@ -119,6 +123,7 @@ func Execute() error {
 func init() {
 	rootCmd.PersistentFlags().StringVar(&host, "host", "http://localhost", "host where Bytebase backend is accessed from, must start with http:// or https://. This is used by Bytebase to create the webhook callback endpoint for VCS integration")
 	rootCmd.PersistentFlags().IntVar(&port, "port", 80, "port where Bytebase backend is accessed from. This is also used by Bytebase to create the webhook callback endpoint for VCS integration")
+	rootCmd.PersistentFlags().IntVar(&datastorePort, "datastore-port", 8081, "port of datastore instance for storing Bytebase data")
```

Comment:
Does go have a library to find a random unused port? If so, I would choose that approach. Because database is considered an implementation detail and there is no need to expose it.

Reply:
We may want a fixed port otherwise we may take up a port that users want to use for other applications. My thought comes from allocated IP range.
https://cloud.google.com/vpc/docs/configure-private-services-access#allocating-range

Reply:
I would see there exists customer really complain about this and understand their usage. Most of the time, hiding the database port leads a better developer experience. I am not willing to sacrifice this for the sake of potential port conflict. Since we are always choosing a random port, if that port is really needed by another app, customer can always workaround this by stopping bytebase and restart again.

Reply:
Unix domain socket without port is the best choice but embedded postgres doesn't give such option :-(.
https://github.com/fergusstrange/embedded-postgres/blob/master/config.go#L12

I think enterprise/serious users will like consistent behavior, and it's easier to debug with a consistent port. A good compromise is to use port + 1 if unspecified and fail early if in-use.

Reply:
port + 1 SGTM

Reply:
Done.